.github/scripts/generate_release_notes.py

```diff
@@ -14,10 +14,10 @@
 
 from github import Github
 
-milestoneProjectRegex = "^(.*) Milestone$"
+milestoneProjectRegex = "^(.*) Milestone .*$"
 releaseNoteRegex = "^RELEASE NOTE:(.*)$"
 dashboardReleaseVersionRegex = "v([0-9\.]+)-?.*"
-majorReleaseRegex = "^([0-9]+\.[0-9]+)\.[0-9]+$"
+majorReleaseRegex = "^([0-9]+\.[0-9]+)\.[0-9]+.*$"
```

Comment:
Works for projects titled `0.0.0-beta.1 Milestone 1`
Creates release notes `0.0.0-beta.1.md` with env vars`REL_BRANCH=0.0` and `REL_VERSION=0.0.0-beta.1`